packages/devextreme/scss/widgets/generic/menu/_index.scss

```diff
@@ -78,6 +78,11 @@
       @include dx-icon-font-centered-sizing($generic-base-icon-size);
     }
   }
+
+  .dx-menu-item-has-icon.dx-state-focused,
+  .dx-menu-item-has-text.dx-state-focused {
```

Comment:
please do it exactly how it's done in material. We wanna this styles be applied only when item is expanded

Reply:
@jdvictoria  i found that we already have necessary styles in menubase css:
```javascript
.dx-menu-item-expanded {
  color: $menu-item-expanded-color;
  background-color: $menu-item-hover-bg;
}
```
Let's just update the selector to make it applying when item is focused or hovered